src/utilities/getIndicatorTableData.js

```diff
@@ -75,16 +92,26 @@ const getIndicatorTableColumns = (indicator, period) => {
   const descriptionColumn = COLUMNS.DESCRIPTION;
   const codeColumn = COLUMNS.CODE;
   // eslint-disable-next-line no-unused-vars
-  const valueColumns = indicator.columns.map(({ description, code, valueType }) => ({
+  const valueColumns = indicator.columns.map(({ description, code }) => ({
```

Comment:
`indicator.columns.map(({ description }` and get rid of the eslint disable above?

Reply:
Was just being used as a placeholder here. Fixed in #1911 (why I suggested you might want to review them in that order :rofl:).